clients/include/blas2/testing_hbmv.hpp

```diff
@@ -80,65 +82,75 @@ void testing_hbmv_bad_arg(const Arguments& arg)
-                HIPBLAS_STATUS_INVALID_VALUE);
-            EXPECT_HIPBLAS_STATUS(
-                hipblasHbmvFn(handle, uplo, N, K, alpha, dA, lda, dx, incx, nullptr, dy, incy),
-                HIPBLAS_STATUS_INVALID_VALUE);
+            DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
+                        hipblasHbmvFn,
+                        (handle, uplo, N, K, nullptr, dA, lda, dx, incx, beta, dy, incy));
+
+            DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
+                        hipblasHbmvFn,
+                        (handle, uplo, N, K, alpha, dA, lda, dx, incx, nullptr, dy, incy));
 
             if(pointer_mode == HIPBLAS_POINTER_MODE_HOST)
             {
                 // For device mode in rocBLAS we don't have checks for dA, dx, dy as we may be able to quick return
-                EXPECT_HIPBLAS_STATUS(
-                    hipblasHbmvFn(
-                        handle, uplo, N, K, alpha, nullptr, lda, dx, incx, beta, dy, incy),
-                    HIPBLAS_STATUS_INVALID_VALUE);
-                EXPECT_HIPBLAS_STATUS(
-                    hipblasHbmvFn(
-                        handle, uplo, N, K, alpha, dA, lda, nullptr, incx, beta, dy, incy),
-                    HIPBLAS_STATUS_INVALID_VALUE);
-                EXPECT_HIPBLAS_STATUS(
-                    hipblasHbmvFn(
-                        handle, uplo, N, K, alpha, dA, lda, dx, incx, beta, nullptr, incy),
-                    HIPBLAS_STATUS_INVALID_VALUE);
+                DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
+                            hipblasHbmvFn,
+                            (handle, uplo, N, K, alpha, nullptr, lda, dx, incx, beta, dy, incy));
+
+                DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
+                            hipblasHbmvFn,
+                            (handle, uplo, N, K, alpha, dA, lda, nullptr, incx, beta, dy, incy));
+
+                DAPI_EXPECT(HIPBLAS_STATUS_INVALID_VALUE,
+                            hipblasHbmvFn,
+                            (handle, uplo, N, K, alpha, dA, lda, dx, incx, beta, nullptr, incy));
             }
 
             // With alpha == 0 can have A and x nullptr
-            CHECK_HIPBLAS_ERROR(hipblasHbmvFn(
-                handle, uplo, N, K, zero, nullptr, lda, nullptr, incx, beta, dy, incy));
+            DAPI_CHECK(hipblasHbmvFn,
+                       (handle, uplo, N, K, zero, nullptr, lda, nullptr, incx, beta, dy, incy));
 
             // With alpha == 0 && beta == 1, all other ptrs can be nullptr
-            CHECK_HIPBLAS_ERROR(hipblasHbmvFn(
-                handle, uplo, N, K, zero, nullptr, lda, nullptr, incx, one, nullptr, incy));
+            DAPI_CHECK(hipblasHbmvFn,
+                       (handle, uplo, N, K, zero, nullptr, lda, nullptr, incx, one, nullptr, incy));
+        }
+
+        // Check 64-bit API with quick return
+        if(arg.api & c_API_64)
+        {
+            DAPI_CHECK(hipblasHbmvFn, (handle, uplo, N, K, zero, dA, lda, dx, incx, one, dy, incy));
```

Comment:
Did you mean to test 64-bit sizes here or no?

Reply:
I don't mind adding it but thought I could do it after the memory allocation is fixed or else we might see some intermittent failures.

Reply:
> Did you mean to test 64-bit sizes here or no?

Not the sizes just the 64-bit API call them and see if returns status success.